### ISSUE TITLE
send_ir ignores HTTP error

### ISSUE THREAD

**Issue by yoichi** [yoichi/remo_util#2]
send_ir does not handle error on requests.post()

**Comment by yoichi** [comment]
```
$ curl -i http://192.168.1.9/messages -H "X-Requested-With: requests" -d `python -m aeha 0x0220 0xb0 0x50 0x3d 0xdd -r 4`
HTTP/1.0 400 Bad Request
Server: Remo/1.0.62-gabbf5bd
Content-Type: text/plain
```

**Comment by yoichi** [comment]
send same request body with send_ir

```
diff --git a/aeha.py b/aeha.py
index a050ff0..f2ba96b 100644
--- a/aeha.py
+++ b/aeha.py
@@ -31,7 +31,7 @@ if __name__ == '__main__':
     'compose IR signal in AEHA format'
     parser = argparse.ArgumentParser(
         description='compose IR signal for SONY remote controller')
-    parser.add_argument('-r', '--repeat', type=int, default=2)
+    parser.add_argument('-r', '--repeat', type=int, default=4)
     parser.add_argument('-t', '--trailer', type=int, default=75000)
     parser.add_argument('vendor', help='vendor code')
     parser.add_argument('command', nargs='*')
diff --git a/send_ir.py b/send_ir.py
index 5b0aa48..ed0f894 100644
--- a/send_ir.py
+++ b/send_ir.py
@@ -8,9 +8,10 @@ import requests
 def send_ir(ip_address, signal_format, rest_args):
     data = subprocess.check_output(
         [sys.executable, '-m', signal_format] + rest_args)
-    requests.post('http://{}/messages'.format(ip_address),
-                  data=data,
-                  headers={'X-Requested-With': 'requests'})
+    r = requests.post('http://{}/messages'.format(ip_address),
+                      data=data,
+                      headers={'X-Requested-With': 'requests'})
+    print(r)
 
 
 if __name__ == '__main__':
```

```
$ python send_ir.py 192.168.1.9 aeha 0x0220 0xb0 0x50 0x3d 0xdd
<Response [200]>
```

It works unexpectedly...

**Comment by yoichi** [comment]
After investigating with tcpdump, I found 400 error with curl is caused by the Expect header sent by curl. It can be avoided by specifying empty value by -H "Expect:".

Ref: https://gms.tf/when-curl-sends-100-continue.html

**Comment by yoichi** [comment]
I've implemented error handling in cea0f33
